man/vconsole.conf.xml

```diff
@@ -67,6 +67,7 @@
     shell compatible execution engine.</para>
 
     <para>Note that the kernel command line options
+    <varname>vconsole.n_setup_vts=</varname>,
```

Comment:
the code calls this "vconsole.n.setup_vts" currently. I think the man page is right, and the code should be changed (see above)